core/rawdb/schema.go

```diff
@@ -66,6 +66,7 @@ var (
 	bloomBitsPrefix       = []byte("B") // bloomBitsPrefix + bit (uint16 big endian) + section (uint64 big endian) + hash -> bloom bits
 	SnapshotAccountPrefix = []byte("a") // SnapshotAccountPrefix + account hash -> account trie value
 	SnapshotStoragePrefix = []byte("o") // SnapshotStoragePrefix + account hash + storage hash -> storage trie value
+	CodePrefix            = []byte("c") // CodePrefix + account hash -> account code
```

Comment:
Feedback wanted: Should we assign the prefix with a character out the scope of `0-9a-f`? In this way, we won't mix the code with trie nodes

Reply:
The hashes are stored as bytes in the key, not hex encoded, so there is no character that doesn't collide. `c` is fine, we'll need to prefix the trie nodes too eventually.

Reply:
Ah, I see